docs/docs/constraints.md

```diff
@@ -28,108 +28,108 @@ Regex is allowed for LIKE and UNLIKE operators; to match ANY value, use the stri
 
 ## Operators
 
-### UNIQUE operator
+### CLUSTER operator
+**Value** (required): A rack ID, set of rack IDs, or a specific node.
```

Comment:
The value of the attribute. This does not have to be that rack id (it is just an example)